themes/digital.gov/layouts/guides/card-guide.html

```diff
@@ -0,0 +1,14 @@
+<article class="dg-guides-card">
+  <div class="dg-guides-card__content">
+    <span class="dg-guides-card__kicker">Digital.gov Guide</span>
+    <h2 class="dg-guides-card__title">
+      <a href="{{ .Permalink | relURL }}">{{ .Title }}</a>
```

Comment:
@mejiaj I found a way to make the entire card a link but it requires an `<a>` tag that isn't semantic. USWDS documentation for `usa-card` doesn't provide any guidance on this pattern.

Reply:
We can consider this a future enhancement.

Once we take this one we can contribute it back to the design system. You can see an example here [Inclusive components - Redundant click](https://inclusive-components.design/cards/#:~:text=pseudo%2Dcontent%20trick-,The%20redundant%20click%20event,-Alternatively%2C%20we%20can)

Reply:
Will create card in the backlog to capture this for additional card refactoring. It's all ready implemented for `news` and `event` cards.